Modules/@babylonjs/react-native/shared/Shared.h

```diff
@@ -16,3 +17,14 @@ inline Napi::FunctionReference GetFlushedQueue(Napi::Env env)
     auto flushedQueue{ batchedBridge.Get("flushedQueue").As<Napi::Function>() };
     return Napi::Persistent(flushedQueue);
 }
+
+// On iOS, directly calling flushedQueue breaks some kind of internal UI state and we start getting errors like:
+// *** Assertion failure in -[RCTNativeAnimatedNodesManager disconnectAnimatedNodes:childTag:](), node_modules/react-native/Libraries/NativeAnimation/RCTNativeAnimatedNodesManager.m:138
+// [native] Exception thrown while executing UI block: 'parentNode' is a required parameter
+// However, calling setTimeout eventually also calls flushedQueue, but doesn't result in the above error,
+// so we'll go this route until we (hopefully) get a better resolution from the React Native team at Facebook.
+inline std::shared_ptr<facebook::jsi::Function> GetSetTimeout(facebook::jsi::Runtime& rt)
+{
+    auto code{std::make_shared<const facebook::jsi::StringBuffer>("() => setTimeout(() => {})")};
+    return std::make_shared<facebook::jsi::Function>(rt.evaluateJavaScript(std::move(code), "").asObject(rt).asFunction(rt));
```

Comment:
Can/should you use napi here instead of jsi?

Reply:
I would assume not? Why bring in an extra layer where we don't need the napi abstraction?

Reply:
I used napi in the other function..

Reply:
We can revisit this if you want to talk about it more, or we can wait and see if we get a better solution in the Discord thread.